src/pages/dashboardUserEdit/components/AddLegalRepresentativeForm.tsx

```diff
@@ -61,40 +61,45 @@ export default function AddLegalRepresentativeForm({
 }: Readonly<LegalRepresentativeProps>) {
   const [isChangedManager, setIsChangedManager] = useState(false);
   const [dynamicDocLink, setDynamicDocLink] = useState<string>('');
-
   const [previousLegalRepresentative, setPreviousLegalRepresentative] =
-    useState<ProductUserResource>();
+    useState<ProductUserResource | null>();
+
   const requestId = uniqueId();
   const { t } = useTranslation();
   const setLoadingCheckManager = useLoading(LOADING_TASK_CHECK_MANAGER);
   const setLoadingGetLegalRepresentative = useLoading(LOADING_TASK_GET_LEGAL_REPRESENTATIVE);
   const addError = useErrorDispatcher();
 
+  const findNewestManager = (managers: Array<ProductUserResource>): ProductUserResource | null => {
+    if (!managers || managers.length === 0) {
+      return null;
+    }
+
+    return managers.reduce((newest, current) => {
+      if (!newest.createdAt || !current.createdAt) {
+        return newest;
+      }
+
+      return new Date(current.createdAt) > new Date(newest.createdAt) ? current : newest;

```

Comment:
`createdAt` property is already a Date. Maybe we can refactor this line into:
```suggestion
      return current.createdAt.getTime() > newest.createdAt.getTime() ? current : newest;

```

Reply:
removed unnecessary date conversion